docker-compose.yml

```diff
@@ -76,7 +76,7 @@ networks:
       name: hashicorp
   qkm:
     external:
-      name: deps_qkm
+      name: qkm
```

Comment:
same here, no changes needed for this PR

Reply:
this is fine

Reply:
darioAnongba is right, I forgot the docker-compose up will prepend the project/dir name.